src/raylib.h

```diff
@@ -692,7 +692,7 @@ extern "C" {            // Prevents name mangling of functions
 //------------------------------------------------------------------------------------
 
 // Window-related functions
-RLAPI void InitWindow(int width, int height, void *data);         // Initialize window and OpenGL context
+RLAPI void InitWindow(int width, int height, const char *title);         // Initialize window and OpenGL context
```

Comment:
Maybe write some ifdef for concrete

Reply:
Yes in raylib.h

Reply:
I just removed it to avoid PLATFORM definition in raylib.h and make it generic for all platforms. I know that `void *data` it's not the most elegant solution but for now I think is the best approach.

Reply:
I see it as small security hole.

Reply:
You look to commit 80b8bd769faddf0c1381502ac171f8d82d837ccc